.ci/run-link-check-plugin.sh

```diff
@@ -14,7 +14,7 @@ echo "------------ grep of linkcheck.html--BEGIN"
 # grep could return exit code 1 if nothing is matching
 grep externalLink target/site/linkcheck.html | cat
 echo "------------ grep of linkcheck.html--END"
-RESULT=$(grep externalLink target/site/linkcheck.html | grep -v 'Read timed out' | wc -l)
+RESULT=$(grep externalLink target/site/linkcheck.html | grep -c -v 'Read timed out' || true)
```

Comment:
I think it is better to permanently disable this validation, a code become more cryptic, with not much benefits, perofmance benefits that this approach provide is notthins for us here, we  more value ease to read of scripts by people.

Reply:
@nrmancuso , do you agree ?

Reply:
@romani agreed, @shubh220922 please update PR to have no changes in script, and explanation above `disabled` like:
```bash
# permanently disabled, intention of 'wc -l' is more obvious than '-c'  grep option, and 'wc -l' mitigates failure of grep to find results
```

Reply:
in PR should be only comment in rc file
`# permanently disabled,  'grep -c || wc -l' is easier to read than 'grep -c || true'

Reply:
@romani @nrmancuso got it. PR incoming.

Reply:
We should not include `-c` in first grep example at https://github.com/checkstyle/checkstyle/pull/12751#discussion_r1115752989

Reply:
@nrmancuso ok so the comment looks more like:

``# SC2126 permanently disabled,  'grep | wc -l' is easier to read than 'grep -c || true'``

Reply:
@shubh220922 yes, first one is pipe to wc, second is bash OR.